org.civicrm.hremergency/xml/services.xml

```diff
@@ -0,0 +1,8 @@
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="emergency_contact.service" class="CRM_Hremergency_Service_EmergencyContactService"/>
```

Comment:
If I saw `hremergency.emergency_contact` I would think it sounds like the entity itself. I prefer the style of `<entity>.<service_type>` like `contact.deduper` or `email.validator`. The fact that the class is EmergencyContactService could be disputed because it's overgeneral, but in this case I didn't have any other more specialized services for emergency contacts so thought I would start with this one which could be modified and split up later.

Reply:
If you really think about it, it doesn't matter if it's a service, an entity, a validator or anything else. What matters is that I will get an object and that I can use it. I'm not sure if `<entity>.<service_type>` is a good approach. No all services/validators/etc have to be related to an entity.

Anyway, I don't think this is such a big problem right now. We only have 2 or 3 services like this in the entire application. Let's keep the ID we have now and, as you said, we can modify things later, if necessary.